DashAI/back/core/schema_fields/int_field.py

```diff
@@ -0,0 +1,46 @@
+from typing import Optional
+
+from pydantic import Field
+from typing_extensions import Annotated
+
+
+def int_field(
+    description: str,
+    default: int,
+    minimum: Optional[int] = None,
+    maximum: Optional[int] = None,
```

Comment:
I think it would be a good idea here to keep the pydantic notation of ge, gt, le, lt:
https://docs.pydantic.dev/latest/concepts/fields/#numeric-constraints

Also, it would be an equally good idea to add the greater than and less than.


```suggestion
    ge: Optional[int] = None,
    gt: Optional[int] = None,
    le: Optional[int] = None,
    lt: Optional[int] = None,
```

Reply:
Ok, I used minimum and maximum to follow the names used in the schematic, but it is more explicit this way.